inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/ParseWikidata.java

```diff
@@ -489,6 +505,21 @@ private void analyzeLexeme(Lexeme lexeme) {
                     throw new IllegalArgumentException(lexeme.lexicalCategory + " is not a known part of speech grammeme for " + lexeme.id + "(" + lemma.value + ")");
                 }
             }
+            int qVariantIdx = currentLemmaLanguage.indexOf(VARIANT_SEPARATOR);
+            if (qVariantIdx >= 0) {
+                // The languages can have wierd Q entry after the desired language.
+                // A spelling variant is informative. Most of the rest are irrelevant.
+                var additionalCategory = currentLemmaLanguage.substring(qVariantIdx + VARIANT_SEPARATOR.length());
+                currentLemmaLanguage = currentLemmaLanguage.substring(0, qVariantIdx);
+                var variant = Grammar.getMappedGrammemes(additionalCategory);
+                if (variant == null) {
+                    if (parserOptions.debug) {
+                        System.err.println("Line " + lineNumber + ": " + additionalCategory + " is not a known grammeme for the language variant " + lexeme.id + "(" + lemma.value + ")");
+                    }
+                    continue;
+                }
+                lemma.grammemes.addAll(variant);
+            }
```

Comment:
This chunk helps with ignoring some of the variants. This might need to be extended further for the inflections.